cpp_template_pkg/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>cpp_template_pkg</name>
+  <version>0.0.0</version>
+  <description>A template package used to demonstrate ROS2 GitHub Actions</description>
+  <maintainer email="foo@bar.baz">Foobar</maintainer>
+  <license>Apache 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <test_depend>ament_copyright</test_depend>
```

Comment:
Do you need copyright and common, or can you just pull auto and get all others recursively?